script1.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-unused-vars */
+const nameInput = document.getElementById('nameInput');
+const authorInput = document.getElementById('authorInput');
+const addBtn = document.querySelector('.add-btn');
+const storageKey = 'booksLocalData';
+const loadedBooks = JSON.parse(localStorage.getItem(storageKey)) || [];
+const bookslist = document.querySelector('.books-list');
+
+function loadBook() {
+  bookslist.innerHTML = '';
+  if (loadedBooks != null) {
+    for (let i = 0; i < loadedBooks.length; i += 1) {
+      bookslist.innerHTML += `<div class="book-info">
+              <p>${loadedBooks[i].name}</p>
+              <p>${loadedBooks[i].author}</p>
+              <button class="remove-btn" id="btn-${i}" onclick="deleteBook(${i})" value="remove">Remove</button>
+              <hr>
+              </div>`;
+    }
+  }
+}
+
+window.onload = () => {
+  loadBook();
+};
+
+function addBook() {
+  const book = { name: nameInput.value, author: authorInput.value };
```

Comment:
- [ ] [OPTIONAL] You have done well creating the book object here. Please make sure that the book object has a unique `id` attribute as this could help you when deleting book items with duplicate values 👍